tutorials/plot_background_statistics.py

```diff
@@ -0,0 +1,642 @@
+#
+# .. rst-class:: skinnytable
+#
+#   +----------+--------+------------------+------------------+
+#   |                   |          Null hypothesis            |
+#   |                   +------------------+------------------+
+#   |                   |       True       |       False      |
+#   +==========+========+==================+==================+
+#   |          |        | Type I error     | Correct          |
+#   |          | Yes    |   False positive |   True positive  |
+#   + Reject   +--------+------------------+------------------+
+#   |          |        | Correct          | Type II error    |
+#   |          | No     |   True Negative  |   False negative |
+#   +----------+--------+------------------+------------------+
+#
+# To see why, consider a standard :math:`\alpha = 0.05`.
+# For a single test, our probability of making a type I error is 0.05.
+# The probability of making at least one type I error in
+# :math:`N_{\mathrm{test}}` independent tests is then given by
+# :math:`1 - (1 - \alpha)^{N_{\mathrm{test}}}`:
+
+N = np.arange(1, 80)
+alpha = 0.05
+p_type_I = 1 - (1 - alpha) ** N
+fig, ax = plt.subplots(figsize=(4, 3))
+ax.scatter(N, p_type_I, 3)
+ax.set(xlabel='$N_{\mathrm{test}}$', ylabel='Probability of ≥ 1\ntype I error',
+       xlim=N[[0, -1]], ylim=[0, 1])
+ax.grid(True)
+fig.tight_layout()
+mne.viz.utils.plt_show()
+
+###############################################################################
+# To combat this problem, multiple methods exist. Typically these
+# provide control over either the:
+#
+# 1. `Familywise error rate (FWER) <fwer>`_
+#      The probability of making one or more type I errors:
+#
+#      .. math::
+#        \mathrm{P}(N_{\mathrm{type\ I}} >= 1 | H_0)
+#
+# 2. `False discovery rate (FDR) <fdr>`_
+#      The expected proportion of rejected null hypotheses that are
+#      actually true:
+#
+#      .. math::
+#        \mathrm{E}(N_{\mathrm{type\ I}} / N_{\mathrm{reject}}
+#        | N_{\mathrm{reject}} > 0)
+#        \mathrm{P}(N_{\mathrm{reject}} > 0 | H_0)
+#
+# We cover some techniques that control FWER and FDR below.
+#
+# Bonferroni correction
+# ^^^^^^^^^^^^^^^^^^^^^
+# Perhaps the simplest way to deal with multiple comparisons, `Bonferroni
+# correction <https://en.wikipedia.org/wiki/Bonferroni_correction>`_
+# conservatively multiplies the p-values by the number of comparisons to
+# control the FWER.
+
+titles.append('Bonferroni')
+ts.append(ts[-1])
+ps.append(bonferroni_correction(ps[0])[1])
+mccs.append(True)
+plot_t_p(ts[-1], ps[-1], titles[-1], mccs[-1])
+
+###############################################################################
+# False discovery rate (FDR) correction
+# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+# Typically FDR is performed with the Benjamini/Hochberg procedure, which
+# is less restrictive than Bonferroni correction for large numbers of
+# comparisons (fewer type II errors) but provides less strict control of
+# errors (more type I errors).
```

Comment:
My conceptual understanding of FWER v FDR is this: you use FWER if you are doubting there is any effect of your manipulation at all, and you want a strict error rate control on the claim that your manipulation makes a difference, any difference, whatsoever. You use FDR if you assume there probably is something going on, and you want a conservative estimate of where and when these effects appear.

If these are accurate, I like them better than how it is sometimes stated - that one is "more conservative" than the other. That is factually true, but seems to me to miss the categorical distinction.

Reply:
That sounds reasonable to me, but I'd rather bother a statistician about this more colloquial interpretation to gain confidence before adding it (so probably in a follow-up PR if we want it).

Reply:
Yeah we can do it like this, and then later I can try forcing people to be super anal about p values :D